src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+}
+
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  padding: 50px 40px;
+  gap: 46px 48px;
+  justify-content: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --column-count: 4;
+  }
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+  position: relative;
+  box-sizing: border-box;
+  padding-left: 16px;
+  padding-right: 16px;
+  transition: all 300ms ease;
+
+  &__imac-image {
+    margin: 32px 0;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 8px;
+    margin-bottom: 0;
+    text-decoration: none;
+    color: $main-accent;
+    display: flex;
+    line-height: 18px;
+  }
+
+  &__code {
+    display: flex;
+    margin-top: 4px;
+    font-size: 10px;
+    color: $secondary;
+    line-height: 14px;
+    width: 100%;
+  }
+
+  &__ratings {
+    margin-top: 8px;
+    display: flex;
+    width: 100%;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    color: $main-accent;
+    margin-left: auto;
+    margin-top: 0;
+    margin-bottom: 0;
```

Comment:
```suggestion
    margin: 0;
```

Reply:
Not fixed 
<img width="641" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/b8e78215-0511-492b-8bf6-83d096071622">

Reply:
not fixed